docs/source/science/ATL08/ATL08.ipynb

```diff
@@ -0,0 +1,1386 @@
+{
```

Comment:
<p>If this is DEM, then <a href="https://docs.maap-project.org/en/latest/technical_tutorials/access/accessing_cod.html#Accessing-Cloud-Optimized-Data" target="_blank">https://docs.maap-project.org/en/latest/technical_tutorials/access/accessing_cod.html#Accessing-Cloud-Optimized-Data</a> can be referred for visualizing DEM properly. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/362/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Thanks, I'll take a look! I think we had tried a more proper visualization for a DEM, but it threw an error saying the data wasn't 2D </p>